history/storage.py

```diff
@@ -116,22 +137,22 @@ def retrieve_response(self, spider, request):
 
         data = json.loads(data_string)
 
-        metadata         = data['metadata']
-        request_headers  = Headers(data['request_headers'])
-        request_body     = data['request_body']
+        metadata = data['metadata']
+        # request_headers = Headers(data['request_headers'])
+        # request_body = data['request_body']
         response_headers = Headers(data['response_headers'])
-        response_body    = data['response_body']
+        response_body = data['response_body']
 
-        if 'binary' in data and data['binary'] == True:
+        if 'binary' in data and data['binary'] is True:
             logger.debug('S3Storage: retrieved binary body')
```

Comment:
With a proper logger initialization you shouldn't need to namespace your messages with `S3Storage`.